app/views/records/_records_list.html.erb

```diff
@@ -31,7 +31,13 @@
     <% else %>
       <% @records.each do |r| %>
         <tr>
-          <td><%= link_to full_name(r), record_path(:id => r.id), class: 'navbar-item' %></td>
+          <% if @task %>
+            <td><%= link_to full_name(r), record_path(:id => r.id, :task_id => @task.id), class: 'navbar-item' %></td>
+          <% elsif @product_test %>
+            <td><%= link_to full_name(r), record_path(:id => r.id, :product_test_id => @product_test.id), class: 'navbar-item' %></td>
+          <% else %>
+            <td><%= link_to full_name(r), record_path(:id => r.id), class: 'navbar-item' %></td>
+          <% end %>
```

Comment:
it looks like i'd left an erroneous `.navbar-item` class there... would you mind removing it